### ISSUE TITLE
How to ingest Type references in attributes?

### ISSUE THREAD

**Issue by franklinatportland** [Azure/Purview-Samples#20]
Hi there,
please point me to the right place if this is no the one.

We are trying to load metadata into Purviews Data Catalog via the API (Apache Atlas API) and in our use case there are attributes that reference other types. For example: `Company.addresses[]` where addresses is an Array of Address type.  

The API presents multiple ways to create relationships, but none seem to be able to display the "address" in the list of Company attributes, in the Purview UI.
Basically, we would like to open the Company Entity in the Purview UI and be able to visualize all attributes both primitive types and complex types as the example above.

Any tips are appreciated.
Thanks
